sentry_plugin/hooks/sentry_hook.py

```diff
@@ -70,27 +93,20 @@ class SentryHook(BaseHook):
     """
 
     def __init__(self, sentry_conn_id=None):
-        integrations = []
         ignore_logger("airflow.task")
+        ignore_logger("airflow.jobs.backfill_job.BackfillJob")
         executor_name = configuration.conf.get("core", "EXECUTOR")
 
         sentry_flask = FlaskIntegration()
+        integrations = [sentry_flask]
 
         if executor_name == "CeleryExecutor":
             from sentry_sdk.integrations.celery import CeleryIntegration
+            from sentry_sdk.integrations.tornado import TornadoIntegration
```

Comment:
Do we use tornado exactly whenever we use celery?

Reply:
It shows as a suggestion when I look at the celery errors. On top of that, airflow installs tornado along with Celery.

Reply:
Airflow's Celery doesn't use Tornado. It only uses by the add-on Celery monitoring tool called `Flower` so no need to integrate Tornado unless we need instrumentation too for Flower.

See:
https://docs.celeryproject.org/en/latest/userguide/monitoring.html#flower-real-time-celery-web-monitor
https://airflow.apache.org/cli.html#flower